crates/biome_cli/tests/snapshots/main_cases_editorconfig/should_emit_diagnostics.snap

```diff
@@ -0,0 +1,42 @@
+---
+source: crates/biome_cli/tests/snap_test.rs
+expression: content
+---
+## `.editorconfig`
+
+```editorconfig
+
+[*]
+max_line_length = off
+insert_final_newline = false
+
+```
+
+## `test.js`
+
+```js
+console.log("foo");
+
+```
+
+# Emitted Messages
+
+```block
+configuration ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Key 'insert_final_newline' is incompatible with biome: Biome always inserts a final newline.
```

Comment:
The message should at least tell the user what it should do to fix the issue. E.g. remove the option